atest/acceptance/keywords/elements.robot

```diff
@@ -102,3 +102,7 @@ Get Empty Element Size
     [Tags]  Known Issue Internet Explorer
     ${width}  ${height}=  Get Element Size  id=emptyDiv
     Should be Equal    ${height}    ${0}
+
+Cover Element
+    Cover Element  //img[@src="image.jpg"]
+    Element Should Not be Visible  //img[@src="image.jpg"]
```

Comment:
We should also verify that the `newDiv` is visible. Please also add plank line in the end.